presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,21 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent
+* Fix ST_Buffer for tiny geometries (#13194)
+* Fix ST_Centroid for tiny geometries (#10629)
+* Fix bug that causes failure when reading unpartitioned table from some connectors with session property ``grouped_execution_for_eligible_table_scans`` turned on.
+* Reuse compression buffer in PagesSerde when serializing pages.
```

Comment:
I'm wondering if there is a way to describe user-visible effect of this change. Otherwise, I'd remove this. CC: @yingsu00

Reply:
@yingsu00 *ping*

@neeradsomanchi Let's remove this note.